utils/nvme-cli/Makefile

```
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Luca Barbato
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=nvme-cli
+PKG_VERSION:=2.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=nvme-cli-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/linux-nvme/$(PKG_NAME)/archive/refs/tags/v$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/nvme-cli-$(PKG_VERSION)
+PKG_HASH:=e84bdba276aadcddda8cf5d412e934cc5673af15132ea02180deb5d06af73146
+
+PKG_MAINTAINER:=Luca Barbato <lu_zero@luminem.org>
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/meson.mk
+
+define Package/nvme-cli
+  TITLE:=nvme-cli
+  URL:=https://github.com/linux-nvme/nvme-cli
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+libnvme 
+  SUBMENU:=Filesystem
+  LICENSE:=GPL-2.0-or-later
+  LICENSE_FILES:=LICENSE
```

```suggestion
  SECTION:=utils
  CATEGORY:=Utilities
  SUBMENU:=Filesystem
  TITLE:=NVMe management command line interface
  URL:=https://nvmexpress.org/
  DEPENDS:=+libnvme 
```
sort, for URL, the repo description seems to point to https://nvmexpress.org/